### ISSUE TITLE
Assignment 3-Fibonacci Sequence Generator

### ISSUE THREAD

**Issue by 45bit23** [vtcbca/basic-programming-57bit23#4]
What is Fibonacci Sequence Generator ?

**Comment by 56bit23** [comment]
It is type of program which can obtain fibonacci sequence upto any input integer.

**Comment by 45bit23** [comment]
What is fibonacci sequence?

**Comment by 56bit23** [comment]
It is a type of sequence which starts with 0 and 1.

**Comment by 45bit23** [comment]
Can you give me a hint how the fibonacci sequence is created?

**Comment by 56bit23** [comment]
Yes, the first and second term of sequence is 0 and 1 respectively and the third term will be the addition of first and second term and respectively the fourth term will be addition of second and third term.

**Comment by 45bit23** [comment]
Can you give the first 8 terms of Fibonacci Sequence?

**Comment by 56bit23** [comment]
Yeah here it is
0,1,1,2,3,5,8,11.

**Comment by 45bit23** [comment]
Thank you @56bit23 for your Response!
